docker-compose.yaml

```diff
@@ -24,11 +24,9 @@ services:
       - ./:/app
   db:
     image: postgres:14
-    container_name: integration-tests-prisma
-    restart: always
```

Comment:
Hmm, I don't think we need this line.

We are now removing the Postgres container and other containers on shutdown, which means that the next time Postgres container get starts up, it will come _clean_, see, https://github.com/curiouslearning/education-lrs/pull/3/files#diff-3722d9ba8feb2d3feac8ce71a209a638d4b404e1c53f937188761181594023e2R3-R5